lib/tokenlist.cpp

```diff
@@ -684,7 +684,7 @@ static bool isQualifier(const Token* tok)
 {
     while (Token::Match(tok, "&|&&|*"))
         tok = tok->next();
-    return Token::Match(tok, "{|;");
+    return Token::Match(tok, "{|;|=|throw");
```

Comment:
I am skeptic:
 * why did you include `throw` but skipped other keywords ? `noexcept` is recommended instead of `throw()` and probably much more common. in C code you can have `& throw`.
 * I was hoping that `skipMethodDeclEnding` would skip method qualifiers.